docs/mkdocs/docs/api/macros/nlohmann_json_namespace_begin.md

```diff
@@ -1,38 +1,48 @@
 # NLOHMANN_JSON_NAMESPACE_BEGIN, NLOHMANN_JSON_NAMESPACE_END
 
 ```cpp
-#define NLOHMANN_JSON_NAMESPACE_BEGIN  // (1)
-#define NLOHMANN_JSON_NAMESPACE_END    // (2)
+#define NLOHMANN_JSON_NAMESPACE_BEGIN /* value */  // (1)
+#define NLOHMANN_JSON_NAMESPACE_END   /* value */  // (2)
 ```
 
-These macros can be used to open and close the `nlohmann` namespace. They include an inline namespace used to
-differentiate symbols when linking multiple versions (including different ABI-affecting macros) of this library.
+These macros can be used to open and close the `nlohmann` namespace. They include two inline namespaces used to
+differentiate symbols when linking multiple versions (including different ABI-affecting macros) of this library. See
+['NLOHMANN_JSON_NAMESPACE`](nlohmann_json_namespace.md) for a detailed explanation of the namespace layout.
 
 1. Opens the namespace.
+2. Closes the namespace.
+
+## Default definition
+
+The default definitions open and close the `nlohmann` as well as two inline namespaces. The tag namespace encodes the
+state of ABI-affecting options ([`JSON_DIAGNOSTICS](json_diagnostics.md) and
+[`JSON_USE_LEGACY_DISCARDED_VALUE_COMPARISON`](json_use_legacy_discarded_value_comparison.md)), the version namespace
+the library version number.
```

Comment:
The last part of the sentence feels incomplete.  Maybe something like this?

", and the version namespace encodes the library version number."